A45-retry-stats.md

```diff
@@ -0,0 +1,126 @@
+A55: Exposing OpenCensus Metrics and Tracing for gRPC retry
+----
+* Author(s): dapengzhang0, dfawley, ejona86, markdroth
+* Approver: markdroth
+* Status: In Review
+* Implemented in: C-core, Java
+* Last updated: 2021-08-02
+* Discussion at: https://groups.google.com/g/grpc-io/c/EAbFqaJWp5w
+
+## Abstract
+
+This document is a proposal on how gRPC client would expose OpenCensus metrics and tracing when retry is enbaled. It can be considered as an augment of the [retry gRFC](https://github.com/grpc/proposal/blob/master/A6-client-retries.md).
+
+## Background
+
+There are a collection of OpenCensus [metrics](https://github.com/census-instrumentation/opencensus-specs/blob/master/stats/gRPC.md#measures) for gRPC. When adding retry support to gRPC, we need to clearly define the measurement points for those metrics and add some retry specific metrics to OpenCensus library to present a clearer picture of retry attempts. The retry gRFC discussed exposing [three additional per-method metrics]((https://github.com/grpc/proposal/blob/master/A6-client-retries.md#retry-and-hedging-statistics)) for retry. However, no detail on how to record and expose these metrics was discussed. Also, these metrics still provide very limited information and the histogram view (the third metric) is not very useful because `maxAttempts` is always less than or equal 5. Besides the number of retry attempts, users may also be interested in the total number of transparent retries and the total amount of delay time caused by retry backoff.
```

Comment:
The "three additional per-method metrics" link has too many parentheses and doesn't actually render as a link.

Reply:
Dene.